### PR TITLE
HLM-542 | modify fetch transaction and add post-vc-certify to clickhouse

### PR DESCRIPTION
**Pull request by ajay-beehyv**
@prasanna-egov @varadeth @saiprakash-v

### CODE REVIEW

File: backend/vc-management-service/src/controllers/transaction.controller.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ async function getTransaction(req,res){
         const transactionId = req.params.transactionId;
         const token = req.header("Authorization");
         const response = await sunbirdRegistryService.getTransaction(transactionId,token);
-        res.status(200).json( response.data);
+        const responseMap = {
+            "transactionId": response.data[0].transactionId,
+            "certificateId": response.data[0].certificateId
```
Comment (prasanna-egov) [inline backend/vc-management-service/src/controllers/transaction.controller.js:10]: You need to send out entity, status (Success and Error)

File: backend/vaccination_api/pkg/models/TransactionResponse.go [hunk 2]
```diff
@@ -4,10 +4,7 @@ import "time"
 
 type TransactionResponse struct {
 	TransactionId	string `json:"transactionId"`
-	Osid  			string `json:"osid"`
-	EntityType  	string `json:"entityType"`
-	Message  	 	string `json:"message"`
+	CertificateId  			string `json:"certificateId"`
```
Comment (varadeth) [inline backend/vaccination_api/pkg/models/TransactionResponse.go:7]: please correct identation

**Review by varadeth — CHANGES_REQUESTED** [review]

File: backend/vaccination_api/cmd/analytics_feed/main.go [hunk 3]
```diff
@@ -151,11 +151,8 @@ dt Date
 	_,err = connect.Exec(`
 	CREATE TABLE IF NOT EXISTS vcTransactionEventV1 (
 		transactionId String,
-		entityOsid String,
-		entityType String,
-		message String,
+		certificateId String,
```
Comment (saiprakash-v) [inline backend/vaccination_api/cmd/analytics_feed/main.go:154]: Add entityType as well

File: backend/vaccination_api/cmd/analytics_feed/main.go [hunk 4]
```diff
@@ -555,24 +552,18 @@ func saveVcTransactionEvent(connect *sql.DB, msg string) (string,string, models.
 		tx,_ = connect.Begin()
 		stmt, err = tx.Prepare(`INSERT INTO vcTransactionEventV1 (
 			transactionId ,
-			entityOsid ,
-			entityType ,
-			message ,
+			certificateId ,
```
Comment (saiprakash-v) [inline backend/vaccination_api/cmd/analytics_feed/main.go:555]: Add entity type as well

File: backend/vaccination_api/cmd/analytics_feed/main.go [hunk 5]
```diff
@@ -555,24 +552,18 @@ func saveVcTransactionEvent(connect *sql.DB, msg string) (string,string, models.
...
 	)
 	if err != nil {
 		log.Infof("Error in preparing stmt %+v", err)
 	}
 	if _, err := stmt.Exec(
 		response.TransactionId,
-		response.Osid,
-		response.EntityType,
-		response.Message,
+		response.CertificateId,
```
Comment (saiprakash-v) [inline backend/vaccination_api/cmd/analytics_feed/main.go:565]: Same here

**Review by saiprakash-v — CHANGES_REQUESTED** [review]

**Review by saiprakash-v — APPROVED** [review]

**Review by prasanna-egov — APPROVED** [review]

**Pull request merged**